src/components/EntityPage/File/index.js

```diff
@@ -339,34 +340,37 @@ const FileEntity = ({ api, fileId }) => {
                   tags={file.isLoading || true ? [] : getTags(file.data)}
                 />
               </EntityTitleBar>
-              <EntityActionBar>Share Button</EntityActionBar>
+              <EntityActionBar>
+                <Download
+                  onSuccess={url => {
+                    trackUserInteraction({
+                      category: TRACKING_EVENTS.categories.entityPage.file,
+                      action: 'Download File',
+                      label: url,
+                    });
+                  }}
+                  onError={err => {
+                    trackUserInteraction({
+                      category: TRACKING_EVENTS.categories.entityPage.file,
+                      action: 'Download File FAILED',
+                      label: JSON.stringify(err, null, 2),
+                    });
+                  }}
+                  kfId={data.kf_id}
+                  acl={acl}
+                />
+              </EntityActionBar>
               <EntityContent>
                 <EntityContentSection title="File Properties">
                   <Row style={{ width: '100%' }}>
-                    <Column style={{ flex: 1, paddingRight: 15, border: 1 }}>
-                      <SummaryTable rows={table1} />
-                    </Column>
-                    <Column style={{ flex: 1, paddingLeft: 15, border: 1 }}>
-                      <SummaryTable rows={table2} />
-                    </Column>
+                    <Column style={{ flex: 1, paddingRight: 15, border: 1 }} />
+                    <Column style={{ flex: 1, paddingLeft: 15, border: 1 }} />
                   </Row>
                 </EntityContentSection>
                 <EntityContentDivider />
-                <EntityContentSection title="Associated Participants/Biospecimens">
-                  <BaseDataTable
-                    loading={file.isLoading}
-                    data={particpantBiospecimenData(file.data)}
-                    columns={particpantBiospecimenColumns}
-                  />
-                </EntityContentSection>
+                <EntityContentSection title="Associated Participants/Biospecimens" />
```

Comment:
No content in these?

Reply:
had to remove as workaround for data issue, will be added back when merging